exercises/farewell-workflow/solution/Worker/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Run Worker
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Temporalio.Extensions.Hosting;
+using Temporalio.Farewell.Workflow;
+
+IHost host = Host.CreateDefaultBuilder(args)
+    .ConfigureLogging(ctx =>
+        ctx.AddSimpleConsole().SetMinimumLevel(LogLevel.Information))
+    .ConfigureServices(ctx =>
+        ctx.AddHostedTemporalWorker(
+            clientTargetHost: "localhost:7233",
+            clientNamespace: "default",
+            taskQueue: "farewell-workflow")
+            .AddStaticActivities(typeof(TranslateActivities)) // Add Activities

```

Comment:
```suggestion
            .AddStaticActivities<TranslateActivities>() // Add Activities
```

This is a bite more idiomatic